tox.ini

```diff
@@ -121,43 +108,27 @@ commands=
     coverage run --source='.' manage.py test {posargs}
 
 
-[fast]
-# Configuration values necessary to run unittests without migrations.
-# Note: This is not an env will not run if invoked. Use an invocation of:
-#
-#   tox -e unittest-py{36}-dj{111}-wag{23}-fast
-#
-# To run unit tests.
-setenv=
-    DJANGO_SETTINGS_MODULE=cfgov.settings.test_nomigrations
-
-
-[slow]
-# Configuration values necessary to run unittests with migrations.
-# Note: This is not an env will not run if invoked. Use an invocation of:
-#
-#   tox -e unittest-py{36}-dj{111}-wag{23}-slow
-#
-# To run unit tests.
-setenv=
-    DJANGO_SETTINGS_MODULE=cfgov.settings.test
-
+[current]
+# Configuration necessary for the production versions Python, Django and 
+# Wagtail
+# Note: This is not an env will not run if invoked.
```

Comment:
Could you rephrase this comment (and below)? I'm not entirely sure what this means.

Reply:
Certainly! I've also renamed all of these config sections to `-config` to ensure that the fact that they're configuration values rather than testenvs is clearer.

Reply:
And I've fixed to missing "and", which is critical to that sentence.